### PR TITLE
Declare package visibility to query URL handler apps

### LINKED ISSUE #38375: [RNMobile] Video preview not launching on Android 11
## Description
Trying to launch the preview of a video on a video block doesn't work on Android 11 and presents a `Problem opening the video - No application can handle this request. Please install a Web browser`:

Screenshot 2022-01-31 at 15 39 43

## Step-by-step reproduction instructions
1. Run the native mobile demo app on Android
2. Scroll down to find the video block in the demo content
3. Tap on the video block and then tap on the "Play" button to launch the video preview
4. Notice the "Problem opening the video" error dialog

## Expected behaviour
The video should launch in a web browser to preview.

## Actual behaviour
App responds with a `Problem opening the video - No application can handle this request. Please install a Web browser` dialog.

## WordPress information
- Gutenberg mobile version: v1.64.0+

## Device information
- Device: Pixel 2 XL
- Operating system: Android 11
- WordPress app version: v18.5+

### PR DESCRIPTION
**Pull request by hypest**
## Description
Fix launching the video preview from the video block on Android 11+.

## Testing Instructions
1. Run the native mobile demo app on Android
2. Scroll down to find the video block in the demo content
3. Tap on the video block and then tap on the "Play" button to launch the video preview
4. Notice that the preview opens in a browser (perhaps asking you to download the video file actually)

## Types of changes
* Declare that we'll be querying packages for handlers of the `android.intent.action.VIEW` intent on `http` and `https` schemes.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal). 
- [x] I've updated related schemas if appropriate.

### CODE REVIEW

**Review by mchowning — APPROVED** [review]
Tested this via the demo app and WPAndroid and tapping on the video is now properly triggering the system intent. Thanks for the fix!

I actually don't remember ever using a `<queries>` in the manifest file before either, so thanks for also teaching me about that. 😄

**Comment by hypest** [comment]
Thanks for the review Matt! I'll get [the gutenberg-mobile side PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4527) green too and then merge this.

**Comment by hypest** [comment]
Green now! Will kick off the merge train.

File: packages/react-native-bridge/CHANGELOG.md [hunk 1]
```diff
@@ -2,5 +2,13 @@
...
 
+- [***] Fix launching video preview on Android 11+ [#38377]
+
+
+## 1.70.0
+
 - [**] Fix Android handling of Hebrew and Indonesian translations [#37565]
+
+## 1.70.1
+
```
Comment (fluiddot) [inline packages/react-native-bridge/CHANGELOG.md:13]: As the PR is already merged I wouldn't apply any correction, but just wanted to share that I think the section shouldn't have been incorporated in this PR as we'll do it when merging the betafix `1.70.1`.

**Comment by fluiddot** [comment]
@hypest I'm wondering if after this PR has been merged we should close https://github.com/WordPress/gutenberg/issues/38375, wdyt?

File: packages/react-native-bridge/CHANGELOG.md (hunk 1 above)
Comment (hypest) [reply packages/react-native-bridge/CHANGELOG.md:13]: Ouch, apologies. The version number mentioned for the `iOS 13` item is actually `1.50.0` not `1.70.1` 😞 .

Let me know if you think we should fix that in a separate PR or you are planning to do it as part of the 1.70.1 process indeed @fluiddot , thanks!

Comment (fluiddot) [reply packages/react-native-bridge/CHANGELOG.md:13]: > Let me know if you think we should fix that in a separate PR or you are planning to do it as part of the 1.70.1 process indeed @fluiddot , thanks!

No worries, I'm planning to fix it as part of the release process ([reference](https://github.com/WordPress/gutenberg/pull/38276/commits/d52ae744c15681ecdac521299c0216b8042de022)).

**Comment by hypest** [comment]
> @hypest I'm wondering if after this PR has been merged we should close #38375, wdyt?

Good point, closed it now. Thanks for the ping @fluiddot !

**Comment by ParaskP7** [comment]
ℹ️ This fix made it into the [19.1-rc-3](https://github.com/wordpress-mobile/WordPress-Android/releases/tag/19.1-rc-3).

**Pull request merged**